src/plugins/intel_gpu/include/intel_gpu/primitives/softmax.hpp

```diff
@@ -51,13 +40,12 @@ struct softmax : public primitive_base<softmax> {
     /// @details
     /// Being given a 4-dimensional input, which consists of b,f,y,x dimensions, softmax normalizes data which are divided into multiple independent sets.
     /// Specific behaviour is determined by this parameter, as follows:
-    /// - when set to @link softmax::dimension_t softmax::normalize_x @endlink each input row is normalized independently,
-    /// - when set to @link softmax::dimension_t softmax::normalize_y @endlink each input column is normalized independently,
-    /// - when set to @link softmax::dimension_t softmax::normalize_z @endlink each input z-coordinate is normalized independently,
-    /// - when set to @link softmax::dimension_t softmax::normalize_f @endlink each in-depth vector of input is normalized independently,
-    /// - when set to @link softmax::dimension_t softmax::normalize_fyx @endlink each 3d image within input is normalized independently,
-    /// - when set to @link softmax::dimension_t softmax::normalize_all @endlink everything is normalized,
-    dimension_t dimension;
+    /// - when softmax dimension is set to 0 (b dim) each batch vector of input is normalized independently,
+    /// - when softmax dimension is set to 1 (f dim) each in-depth vector of input is normalized independently,
+    /// - when softmax dimension is set to 2 (y dim) each input column is normalized independently,
+    /// - when softmax dimension is set to 3 (x dim) each input row is normalized independently.
```

Comment:
The description seems to be invalid for 5d case (dim=2 means Z, not Y). So you can either fix it or just remove description at all as there's no any plugin specific for this parameter and we can just refer to the operation spec.

Reply:
Yes, this is a special case for an example. The phrase "Being given a 4-dimensional input.." speaks of this.